commcare_connect/program/models.py

```diff
@@ -15,7 +15,7 @@ class Program(BaseModel):
     currency = models.CharField(max_length=3)
     start_date = models.DateField()
     end_date = models.DateField()
-    organization = models.ForeignKey(Organization, on_delete=models.DO_NOTHING, null=True, blank=True)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
```

Comment:
I don't think we want this to cascade, because we might still want it to show up for the network managers. I think for now lets `PROTECT` and we can always change it later

Reply:
Changed here ([0924f21](https://github.com/dimagi/commcare-connect/pull/385/commits/0924f21617da12af23fd9b877b3f13bc67d70a42))

As in program invite flow this script again got changed so to avoid conflict did this here